tests/e2e/e2e_community_update_params_test.go

```diff
@@ -0,0 +1,168 @@
+package e2e_test
+
+import (
+	"context"
+	"encoding/hex"
+	"time"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+	govv1 "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+
+	"github.com/kava-labs/kava/tests/e2e/testutil"
+	"github.com/kava-labs/kava/tests/util"
+	communitytypes "github.com/kava-labs/kava/x/community/types"
+)
+
+func (suite *IntegrationTestSuite) TestCommunityUpdateParams_NonAuthority() {
+	// ARRANGE
+	// setup kava account
+	funds := ukava(1e5) // .1 KAVA
+	kavaAcc := suite.Kava.NewFundedAccount("community-non-authority", sdk.NewCoins(funds))
+
+	gasLimit := int64(2e5)
+	fee := ukava(200)
+
+	msg := communitytypes.NewMsgUpdateParams(
+		kavaAcc.SdkAddress,
+		communitytypes.DefaultParams(),
+	)
+
+	// ACT
+	req := util.KavaMsgRequest{
+		Msgs:      []sdk.Msg{&msg},
+		GasLimit:  uint64(gasLimit),
+		FeeAmount: sdk.NewCoins(fee),
+		Memo:      "this is a failure!",
+	}
+	res := kavaAcc.SignAndBroadcastKavaTx(req)
+
+	// ASSERT
+	_, err := util.WaitForSdkTxCommit(suite.Kava.Tx, res.Result.TxHash, 6*time.Second)
+	suite.Error(err)
```

Comment:
I'd check for a explicit error here. e2e tests could fail for a lot of reasons but we want to check that it's failing because the authority isn't correct.